pages/stylesheets/all.scss

```diff
@@ -22,6 +22,57 @@ a {
   font-size: 1em !important;
   line-height: 1.2em !important;
 }
+.align-horizontally {
+  position: relative;
+  float: left;
+  clear: none;
+  margin-right: 0em;
+  width:100%;
+}
+
+@media (min-width: 40.0625em){
+.align-horizontally {
+  width:20%;
+}
+}
+
+.horizontal-label{
+  position: relative;
+  float: left;
+  width: 110px;
+  margin-left: 1em;
+  height: 100%;
+  font-weight: bold;
+  display: flex;
+  align-items: center;
+  line-height:44px;
+}
+.align-center{
+  align-items: center;
+}
+.checkbox-list{
+position: relative;
+}
+
+.govuk-checkboxes__label {
+  padding: 10px 10px 5px;
+  font-size: 1rem;
+}
+
+.govuk-checkboxes {
+  width: 85%;
+    float: left;
+    display: block;
+    padding-left:1em;
+}
+
+.AccordionItem_assets-dot__12nY7, .AccordionItem_vulnerabilities-dot__2imGq {
+display:none;
+}
```

Comment:
Does the removal of these dots in `components/Form/AccordionItem/index.js` mean that this styling rule isn't needed?

(if not, I suspect this won't work - these look like react-generated classes and will be different each time)

Reply:
Good spot, if that's the case (different each time)... The dots are no longer required because they originally corresponded to coloured checkboxes

Reply:
I tested it locally, the dots are removed completely in a different file, in this PR. 

That means the styling rule isn't needed, so I've removed it.